modules/orbitsoftBidAdapter.js

```diff
@@ -0,0 +1,229 @@
+import { getBidRequest } from 'src/utils';
+import adaptermanager from 'src/adaptermanager';
+
+let CONSTANTS = require('src/constants');
+let utils = require('src/utils');
+let adloader = require('src/adloader');
+let bidmanager = require('src/bidmanager');
+let bidfactory = require('src/bidfactory');
+let Adapter = require('src/adapter');
+
+let styleParamsToFieldsMap = {
+  'title.family': 'f1', // headerFont
+  'title.size': 'fs1', // headerFontSize
+  'title.weight': 'w1', // headerWeight
+  'title.style': 's1', // headerStyle
+  'title.color': 'c3', // headerColor
+  'description.family': 'f2', // descriptionFont
+  'description.size': 'fs2', // descriptionFontSize
+  'description.weight': 'w2', // descriptionWeight
+  'description.style': 's2', // descriptionStyle
+  'description.color': 'c4', // descriptionColor
+  'url.family': 'f3', // urlFont
+  'url.size': 'fs3', // urlFontSize
+  'url.weight': 'w3', // urlWeight
+  'url.style': 's3', // urlStyle
+  'url.color': 'c5', // urlColor
+  'colors.background': 'c2', // borderColor
+  'colors.border': 'c1', // borderColor
+  'colors.link': 'c6', // lnkColor
+};
+
+let OrbitsoftAdapter;
+OrbitsoftAdapter = function OrbitsoftAdapter() {
+  let baseAdapter = Adapter.createNew('orbitsoft');
+
+  baseAdapter.callBids = function (params) {
+    let bids = params.bids || [];
+
+    if (bids.length === 0) {
+      return;
```

Comment:
this conditional is kind of pointless.  if `bids.length === 0` then the for loop below won't iterate and the function will return anyways